modules/ducktests/tests/ignitetest/services/spark.py

```diff
@@ -103,28 +109,40 @@ def clean_node(self, node):
         node.account.ssh("sudo rm -rf -- %s" % SparkService.SPARK_PERSISTENT_ROOT, allow_fail=False)
 
     def pids(self, node):
-        """Return process ids associated with running processes on the given node."""
         try:
             cmd = "jcmd | grep -e %s | awk '{print $1}'" % self.java_class_name(node)
-            pid_arr = [pid for pid in node.account.ssh_capture(cmd, allow_fail=True, callback=int)]
-            return pid_arr
-        except (RemoteCommandError, ValueError) as e:
+            return list(node.account.ssh_capture(cmd, allow_fail=True, callback=int))
+        except (RemoteCommandError, ValueError):
             return []
 
     def java_class_name(self, node):
+        """
+        :param node: Spark node.
+        :return: Class name depending on node type (master or slave).
+        """
         if node == self.nodes[0]:
             return "org.apache.spark.deploy.master.Master"
-        else:
-            return "org.apache.spark.deploy.worker.Worker"
 
-    def master_log_path(self, node):
+        return "org.apache.spark.deploy.worker.Worker"
+
+    @staticmethod
```

Comment:
Do we really need static methods in our code? I don't see any case in our code where usage of them is required. It just add more sugar without need. Also some code styles consider them evil, e.g. [Google Guide](https://google.github.io/styleguide/pyguide.html#217-function-and-method-decorators).

Reply:
Despite the fact that in mentioned above codestyle staticmethods are not welcome in cod, linters generate warnings about unused `self`. And usage of `@staticmethod` is a preferrable way to mitigate this case. This is so common in my practice and widely accepted in a python's community.